wdfn-server/waterdata/templates/macros/components.html

```diff
@@ -46,12 +46,47 @@ <h3 class="usa-alert__heading">Provisional Data Statement</h3>
     </div>
 {%- endmacro %}
 
-{% macro NetworkComponent(network_cd, extent) -%}
+{% macro NetworkComponent(network_cd, extent, collection, narrative) -%}
     <div class="wdfn-component" data-component="network" data-networkcd="{{ network_cd }}" data-extent="{{ extent }}">
-        <h3>Network Map:</h3>
         <p id="overload-map"></p>
         <div id="network-map"></div>
-        <h3>Link Table:</h3>
+        <div id="network-metadata" class="usa-prose">
+            {% if narrative %}
+                <div>{{ narrative|safe }}</div>
+            {% endif %}
+
+            {% if collection.properties.contactInformation.contactName and collection.properties.contactInformation.contactEmail %}
+            <label class="bold-label" for="contact-info">Network Contact</label><br>
+            <span id="contact-info">
+                <span>{{ collection.properties.contactInformation.contactName }}</span>&nbsp;&nbsp;&nbsp;
+                <span><a href="mailto:{{ collection.properties.contactInformation.contactEmail }}"><i class="fas fa-envelope"></i></a>
+                &nbsp;<a href="mailto:{{ collection.properties.contactInformation.contactEmail }}">{{ collection.properties.contactInformation.contactEmail }}</a></span>
+            </span>
+            {% endif %}
+
+            {% if collection.properties.cooperators %}
+            <div id="network-cooperators">
+                <h3>Cooperators</h3>
+                <div class="grid-container">
+                    <div class="grid-row"> 
+                    {% for cooperator in collection.properties.cooperators %}
+                        <div class="grid-col-12 tablet:grid-col-6 desktop:grid-col-4">
```

Comment:
we try to use the Sass mixins rather than the utility classes where possible in our custom markup: https://designsystem.digital.gov/utilities/layout-grid/#sass-mixins. I would like to see utility classes used here.